src/Resources/config/routes/product_variant.yaml

```diff
@@ -0,0 +1,15 @@
+sylius_admin_ajax_all_product_variants_by_phrase:
```

Comment:
Could we add a comment here saying where this was taken?

Reply:
Added 👍